source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
+    2. Run Mattermost as a desktop app.
 
-`Homebrew <https://brew.sh>`__ users can install with ``brew install --cask mattermost``.
+  .. tab:: Mac 10.9+
 
-Linux
------
+    1. Download the latest version of the Mattermost desktop app:
+      
+      - `Intel systems <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-mac-x64.dmg>`__
+      - `M1 systems <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-mac-m1.dmg>`__ (Beta)
+
+    2. Double-click the download to open the disk image.
+
+    3. Drag the Mattermost application to the **Applications** folder.
+
+    .. tip::
+      
+      `Homebrew <https://brew.sh>`__ users can install with ``brew install --cask mattermost``.
+
+  .. tab:: Windows 8.1+
+
+    1. Download the latest version of the Mattermost desktop app: `32/64-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-setup-5.0.4-win.exe>`__
 
-Generic Linux package
-~~~~~~~~~~~~~~~~~~~~~
+    2. From the **\Downloads** folder, right-click on the file ``mattermost-desktop-setup-5.0.4-win.exe`` and select **Open** to start an installer for the app. Once finished, the Mattermost desktop app opens automatically.
 
-1. Download the latest version of the Mattermost desktop app:
+    **MSI Installer and Group Policies (Beta)**
 
-  - 64-bit systems: `mattermost-desktop-5.0.4-linux-x64.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x64.tar.gz>`__
-  - 32-bit systems: `mattermost-desktop-5.0.4-linux-ia32.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-ia32.tar.gz>`__
+    You can download the latest version of the Mattermost Desktop App MSI installer (Beta):
 
-2. Extract the archive to a convenient location. You can then execute ``mattermost-desktop``, which is located inside the extracted directory.
+    - MSI for `64-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x64.msi>`__
+    - MSI for `32-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x86.msi>`__
 
-3. To create a Desktop launcher, open the file **README.md** and follow the instructions in the **Desktop launcher** section.
+    See the `MSI installer and group policy documentation <https://docs.mattermost.com/install/desktop-msi-gpo.html>`__ for instructions on installing the Mattermost Desktop App via an MSI installer and configuring supported group policies. Changes to group policies require you to restart Mattermost for those changes to take effect.
 
-Ubuntu and Debian-based systems
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    The following group policies are available:
 
-Unofficial, community-driven ``.deb`` packages are available.
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
```

Comment:
Any changes needed here?

Reply:
You can also disable auto-updates using `Enable Auto Updater`

Reply:
@devinbinnie - Can you provide more detail on how to disable auto-updates, please?

Reply:
It's the same procedure that you would do to enable/disable server management, or edit the server list. It's just a boolean switch, where if it's set to `true`, updates are enabled. And if `false`, they are disabled.

Reply:
@cwarnermm So this isn't done in the product, it's generally done by system administrators in the Group Policy Editor. This isn't something that should be changed by the user.

Just to clarify, so we don't presently have any documentation currently with respect to editing the GPO settings? I would have figured we would have since the GPO settings have been around a while. If not, is that something we should be documenting, or is that more on the system administrator side of things?